dev-java/jdbc-postgresql/jdbc-postgresql-9.4_p1204.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
```

Comment:
Did someone forget to run `repoman full` then? I thought it catches stuff like that these days.